recipes/physx/4.x.x/conanfile.py

```diff
@@ -203,9 +212,15 @@ def _configure_cmake(self):
         # Options defined in externals/cmakemodules/NVidiaBuildOptions.cmake
         self._cmake.definitions["NV_APPEND_CONFIG_NAME"] = False
         self._cmake.definitions["NV_USE_GAMEWORKS_OUTPUT_DIRS"] = False
-        if self.settings.compiler == "Visual Studio":
-            self._cmake.definitions["NV_USE_STATIC_WINCRT"] = str(self.settings.compiler.runtime).startswith("MT")
-            self._cmake.definitions["NV_USE_DEBUG_WINCRT"] = str(self.settings.compiler.runtime).endswith("d")
+        if self._is_msvc:
+            if self.settings.compiler == "Visual Studio":
+                is_static_runtime = str(self.settings.compiler.runtime).startswith("MT")
+                is_debug_runtime = str(self.settings.compiler.runtime).endswith("d")
+            else:
+                is_static_runtime = self.settings.compiler.runtime == "static"
+                is_debug_runtime = self.settings.compiler.runtime_type == "Debug"
```

Comment:
I think we need some helper in conan for universal runtime handling, too many recipes have to use complicated conditions like that